HLTrigger/Timer/plugins/ThroughputService.cc

```diff
@@ -85,6 +85,7 @@ void ThroughputService::preGlobalBeginRun(edm::GlobalContext const& gc) {
 
     // define a callback that can book the histograms
     auto bookTransactionCallback = [&, this](DQMStore::IBooker& booker, DQMStore::IGetter&) {
+      auto scope = dqm::reco::DQMStore::IBooker::UseRunScope(booker);
```

Comment:
This is copied from https://github.com/cms-sw/cmssw/blob/c0d007c5a8e169da2cc6dfea20751bd7215fccb9/HLTrigger/Timer/plugins/FastTimerService.cc#L921

Without this call, the harvesting output of the unit test does not contain a `Throughtput` folder, and I'm not really sure why that is.

Reply:
Short version: I think this change is okay. More info below.

I had a look at https://github.com/cms-sw/cmssw/blob/master/DQMServices/Core/README.md, but that didn't really clarify things for me.

If I understand the interface, the DQM scope in `ThroughputService` before the call to
```cpp
auto scope = dqm::reco::DQMStore::IBooker::UseRunScope(booker);
```
corresponds to `scope.oldscope`. The latter returns `1`, which corresponds to `JOB` (and after the call, the scope becomes `RUN`). This seems to match the default set [here](https://github.com/cms-sw/cmssw/blob/d867e73c13e9703c736ceee505fbc14272b58068/DQMServices/Core/interface/DQMStore.h#L556), i.e. `JOB`.

The call to use the `RUN` scope in `FastTimerService` was introduced in #28622 by DQM (maybe `ThoughputService` was simply overlooked in that PR). I don't see a reason why `FastTimerService` and `ThroughputService` should differ in this respect.

With this PR, I managed to produce a ROOT output file with the client `hlt_dqm_clientPB-live_cfg.py` reading `.pb` files produced by re-running a recent HLT menu on 2022 data; in that ROOT output file, I see the `HLT/Throughput` folder, and the plots look as expected. This suggests that this PR does not break the workflow to produce these plots online (which somehow was already working).

Based on the above, I would conclude that this change is okay, even though I don't fully understand it; in particular, I don't know why the `HLT/Throughput` plots were already being produced correctly in the online DQM without this PR.

@cms-sw/dqm-l2 , do you have insight on this?